src/components/MarkerOverlayService.js

```diff
@@ -6,10 +6,30 @@ goog.provide('ga_marker_overlay_service');
   module.provider('gaMarkerOverlay', function() {
 
     this.$get = function(gaStyleFactory) {
-      var overlay, bbox, isAlwaysVisible;
-      var marker = $('<div></div>')
-          .addClass('ga-crosshair')
-          .addClass('marker');
+      var bbox, isAlwaysVisible;
+      var source, layer;
+      var feature = null;
+
+      /** @const */
+      var style = new ol.style.Style({
+        image: new ol.style.Icon({
+          anchor: [0.5, 46],
+          anchorXUnits: 'fraction',
+          anchorYUnits: 'pixels',
+          src: 'img/marker.png'
+        })
+      });
```

Comment:
Usually we define style in `gaStyleFactory` service. See https://github.com/geoadmin/mf-geoadmin3/blob/master/src/components/StylesService.js#L138